### PR TITLE
rails4 compatibility

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

**Comment by johnpmitsch** [comment]
[test]

File: lib/foreman_docker/permitted_attributes.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module Foreman
+  module PermittedAttributes
+    mattr_reader :container_attributes
+
+    @@container_attributes = [:command, :repository_name, :name, :compute_resource_id, :entrypoint,
```
Comment (dLobatog) [inline lib/foreman_docker/permitted_attributes.rb]: I don't understand why this makes a difference?

File: db/migrate/20141024163003_create_docker_registries.rb [hunk 2]
```diff
@@ -3,13 +3,11 @@ def change
     create_table :docker_registries do |t|
       t.string :url
       t.string :name
-      t.integer :id
```
Comment (dLobatog) [inline db/migrate/20141024163003_create_docker_registries.rb]: :+1:

File: lib/foreman_docker/permitted_attributes.rb (hunk 1 above)
Comment (komidore64) [inline lib/foreman_docker/permitted_attributes.rb]: @unorthodoxgeek would you mind elaborating on why [this change](https://github.com/komidore64/foreman-docker/commit/ff1150ccad9e19becde013341e4b0cef17ffbeed) was necessary?

Comment (unorthodoxgeek) [inline lib/foreman_docker/permitted_attributes.rb]: this PR is obsolete and was written for a previous iteration on the rails 4 branch, which is no longer in existence.

**Comment by unorthodoxgeek** [comment]
what is this PR?
I think that the relevant PR is this: https://github.com/theforeman/foreman-docker/pull/126

this PR uses the rails4 code which I first wrote, and uses the strong parameters implementation I started with and we ended up deserting...

**Comment by komidore64** [comment]
I don't particularly care about which pull-request is accepted, just that we get Rails4 compatibility into Foreman-Docker simultaneously or soon after Foreman and Katello become compatible.

@unorthodoxgeek these are the changes that @johnpmitsch, @cpeters, and I have been using for the past few weeks. I didn't mean to step on anyone's toes, it just seemed that your first PR for Rails4 compatibility had become stagnant. Apologies.

**Comment by komidore64** [comment]
alright, closing in favor of #126.

**Pull request closed without merging**